geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/ProtobufOpsProcessor.java

```diff
@@ -58,7 +60,7 @@ public ProtobufOpsProcessor(ProtobufSerializationService serializationService,
       // Don't move to a terminating state for authorization state failures
       logger.warn(e.getMessage());
       result = Failure.of(ProtobufResponseUtilities.makeErrorResponse(e));
-    } catch (ConnectionStateException e) {
+    } catch (EncodingException | DecodingException | ConnectionStateException e) {
```

Comment:
It looks like this PR has already been merged, but I don't think we want to close the connection if an object on the server can't be serialized, or if a client serializes an object wrong.

Reply:
Hmm, I agree with EncodingException but I think poorly serialized data ought to be treated as a fatal error and the connection should be closed.

Reply:
Since what you're describing is the way it used to behave I'll change it to not close the connection.